Source/Scene/Cesium3DTile.js

```diff
@@ -935,6 +936,21 @@ function createPriorityFunction(tile) {
   };
 }
 
+function getJsonContent(arrayBuffer, byteOffset) {
```

Comment:
Something doesn't sit right with me about breaking the symmetry of the `3DTilesContentFactory` interface. 

What if:

1. You move this `getJsonContent()` function into `3DTilesContentFactory`
2. then any time you call `3DTilesContentFactory.gltf()` or `.json()`, the _factory_ calls `getJsonContent()` just before passing things to the content constructor

This way the factory pattern doesn't change at all, but you still get the benefit of moving the JSON parsing code in its own function.

Reply:
One problem is it doesn't know whether to call `3DTilesContentFactory.gltf` or `.json` until it's parsed the JSON

Reply:
Ah I see why you did it this way.